grader_host/config/base.yaml

```diff
@@ -74,9 +74,14 @@ properties:
   resultsQueueUrl:
     default: null
     envVar: RESULTS_QUEUE_URL
+
+  # Timeouts
```

Comment:
Man, who introduced this bespoke config format 😉

Reply:
🤣